pkg/ccl/multitenantccl/tenantcostclient/testdata/consumption

```diff
@@ -732,31 +700,28 @@ Estimated CPU seconds: 0.00
 
 metrics
 ----
-tenant.sql_usage.request_units: 2359.62
-tenant.sql_usage.kv_request_units: 58.62
+tenant.sql_usage.request_units: 2397.61
+tenant.sql_usage.kv_request_units: 70.67
 tenant.sql_usage.read_batches: 4
 tenant.sql_usage.read_requests: 5
 tenant.sql_usage.read_bytes: 2228224
-tenant.sql_usage.write_batches: 5
-tenant.sql_usage.write_requests: 6
-tenant.sql_usage.write_bytes: 11264
+tenant.sql_usage.write_batches: 9
+tenant.sql_usage.write_requests: 10
+tenant.sql_usage.write_bytes: 15410
 tenant.sql_usage.sql_pods_cpu_seconds: 3.82
 tenant.sql_usage.pgwire_egress_bytes: 12345
 tenant.sql_usage.external_io_ingress_bytes: 1024000
 tenant.sql_usage.external_io_egress_bytes: 1024000
-tenant.sql_usage.cross_region_network_ru: 15.61
+tenant.sql_usage.cross_region_network_ru: 41.55
 tenant.sql_usage.estimated_kv_cpu_seconds: 0.00
 tenant.sql_usage.estimated_cpu_seconds: 0.00
-tenant.sql_usage.estimated_replication_bytes: 45056
-tenant.sql_usage.estimated_replication_bytes{from_region="us-central1",from_zone="",to_region="europe-west1",to_zone=""}: 22528
-tenant.sql_usage.estimated_replication_bytes{from_region="us-central1",from_zone="az1",to_region="us-central1",to_zone="az1"}: 11264
-tenant.sql_usage.estimated_replication_bytes{from_region="us-central1",from_zone="az1",to_region="us-central1",to_zone="az2"}: 11264
+tenant.sql_usage.estimated_replication_bytes: 0
 
-# Test multiple requests in the same batch with RU multiplier of 1.
-write count=2 bytes=1024 networkCost=0
+# Test multiple requests in the same batch.
+write count=2 bytes=1024 localities=cross-region
```

Comment:
Hm, do you know why `estimated_replication_bytes` is still 0 despite doing a cross-region write?

Reply:
Because we're only computing `estimated_replication_bytes` when the eCPU model is enabled. When the RU model is enabled, we instead compute `tenant.sql_usage.cross_region_network_ru`. I figured we wouldn't want to compute `estimated_replication_bytes` for Basic clusters.